lib/ui/form/widget/form_survey_question_page.dart

```diff
@@ -61,6 +64,21 @@ class FormSurveyQuestionPage extends StatelessWidget {
     switch (displayType) {
       case DisplayType.smiley:
         return const FormSurveyAnswerSmiley();
+      case DisplayType.heart:
+        return FormSurveyAnswerEmoji(
+          emoji: '❤',
+          totalEmojiCount: _totalEmogiCount,
```

Comment:
Oh, no not like this.

I meant a default value in the constructor of `FormSurveyAnswerEmoji` 💪 

```
  FormSurveyAnswerEmoji({
    Key? key,
    required this.emoji,
    this.totalEmojiCount = 5,
  }) : super(key: key);
```

Reply:
@Shayokh144 Not sure if I missed something here, did we make an update on this? 🙇